pkg/virt-controller/services/renderresources.go

```diff
@@ -690,3 +702,31 @@ func multiplyMemory(mem resource.Quantity, multiplication float64) resource.Quan
 	mem.Add(*additionalOverhead)
 	return mem
 }
+
+func getMemoryLimitsRatio(namespace string, namespaceStore cache.Store) float64 {
+	if namespaceStore == nil {
+		return DefaultMemoryLimitOverheadRatio
+	}
+
+	obj, exists, err := namespaceStore.GetByKey(namespace)
+	if err != nil {
+		log.Log.Errorf("Error retrieving namespace from informer. Using the default memory limits ratio. %s", err.Error())
+		return DefaultMemoryLimitOverheadRatio
+	} else if !exists {
+		log.Log.Errorf("namespace %s does not exist. Using the default memory limits ratio.", namespace)
```

Comment:
Nit: are these really errors if we just happily proceed anyway?

Reply:
Converting them to a warning :)